editor/editor_tests/user_stats_test.cpp

```diff
@@ -0,0 +1,20 @@
+#include "testing/testing.hpp"
+
+#include "editor/user_stats.hpp"
+
+namespace editor
+{
+namespace
+{
+UNIT_TEST(UserStats_Smoke)
+{
+  // This user made only two changes and the possibility of further cahnges is very low.
+  UserStats userStats("Vladimir BI");
```

Comment:
It's not good. Please, use either test http server made by @therearesomewhocallmetim or use mocks.